cilium-dbg/cmd/bpf_multicast_groups.go

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"net/netip"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/common"
+	maps_multicast "github.com/cilium/cilium/pkg/maps/multicast"
+)
+
+var BpfMcastGroupCmd = &cobra.Command{
+	Use:   "group",
+	Short: "Manage the multicast groups.",
+}
+
+func init() {
+	BpfMcastGroupCmd.AddCommand(MulticastGroupListCmd)
+	command.AddOutputOption(MulticastGroupListCmd)
+	BpfMcastGroupCmd.AddCommand(MulticastGroupAddCmd)
+	BpfMcastGroupCmd.AddCommand(MulticastGroupDelCmd)
+}
+
+var MulticastGroupListCmd = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"ls"},
+	Short:   "List the multicast groups.",
+	Long:    "List the multicast groups configured on the node.",
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf multicast group list")
+
+		groupV4Map, err := maps_multicast.OpenGroupV4OuterMap(maps_multicast.GroupOuter4MapName)
+		if err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				fmt.Fprintln(os.Stderr, "Cannot find multicast bpf maps")
+				return
+			}
+
+			Fatalf("Cannot open multicast bpf maps: %s", err)
+		}
+
+		groups, err := groupV4Map.List()
+		if err != nil {
+			Fatalf("Error listing multicast groups: %s", err)
+		}
+
+		if command.OutputOption() {
+			if err := command.PrintOutput(groups); err != nil {
+				Fatalf("error getting output of map in %s: %s\n", command.OutputOptionString(), err)
+			}
+			return
+		}
+		printGroupList(groups)
+	},
+}
+
+func printGroupList(groups []netip.Addr) {
+	// sort groups by address
+	sort.Slice(groups, func(i, j int) bool {
+		return groups[i].Compare(groups[j]) < 0
+	})
+
+	w := tabwriter.NewWriter(os.Stdout, 5, 0, 3, ' ', 0)
+	fmt.Fprintln(w, "Group Address")
+	for _, group := range groups {
+		fmt.Fprintf(w, "%s\n", group.String())
+	}
+	w.Flush()
+}
+
+var MulticastGroupAddCmd = &cobra.Command{
+	Use:   "add <group>",
+	Short: "Add a multicast group.",
+	Long:  "Add a multicast group to the node.",
+	Example: `	Following command adds group 229.0.0.1 to BPF multicast map of the node:
+cilium-dbg bpf multicast group add 229.0.0.1`,
```

Comment:
```suggestion
	Example: `The following command adds group 229.0.0.1 to BPF multicast map of the node:

    cilium-dbg bpf multicast group add 229.0.0.1`,
```

Currently it renders a bit weird in the markdown:
![Screenshot from 2024-03-13 20-40-33](https://github.com/cilium/cilium/assets/3199463/b86419a3-4ec1-4e75-be50-02eb860c66bd)